### PR TITLE
optimize log operation

### PR DESCRIPTION
**Pull request by xiaomuqiao**
issue #62

### CODE REVIEW

**Comment by xiaomuqiao** [comment]
@superxi911  @gaocegege @zoumo  PTAL

File: worker/ci/runner/runner.go [hunk 1]
```diff
@@ -29,6 +29,7 @@ import (
 	docker_client "github.com/fsouza/go-dockerclient"
 )
 
+// BuildStatus represent unsigned int
```
Comment (gaocegege) [inline worker/ci/runner/runner.go]: BuildStatus is the type for status of build
is better, I think.

**Comment by xiaomuqiao** [comment]
ooh, good idea @gaocegege

File: worker/ci/runner/runner.go (hunk 1 above)
Comment (xiaomuqiao) [inline worker/ci/runner/runner.go]: I think too

File: worker/ci/runner/runner.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ import (
 	docker_client "github.com/fsouza/go-dockerclient"
 )
 
-// BuildStatus represent unsigned int
+// BuildStatus represent the type for status of build.
```
Comment (gaocegege) [inline worker/ci/runner/runner.go]: s/represent/is or represents

Comment (xiaomuqiao) [inline worker/ci/runner/runner.go]: ok

File: pkg/osutil/osutil.go [hunk 3]
```diff
@@ -50,7 +50,7 @@ func GetStringEnv(name, def string) string {
 func GetIntEnv(name string, def int) int {
 	val, err := strconv.Atoi(os.Getenv(name))
 	if err != nil {
-		log.Infof("Env variant %s is not numeric, using default value: %s", name, def)
+		log.Infof("Env variant %s is not numeric, using default value: %v", name, def)
```
Comment (superxi911) [inline pkg/osutil/osutil.go]: s/%v/%d

File: worker/log/log.go [hunk 4]
```diff
@@ -29,13 +29,22 @@ import (
...
 
 const (
 	LogSpecialMark string = "@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@"
 	LogReplaceMark string = "--->"
+	// FileBufferSize is the default size of file buffers.
+	FileBufferSize = 64 * 1024 * 1024
+)
+
+var (
+	Output filebuffer.FileBuffer
```
Comment (superxi911) [inline worker/log/log.go]: need comments

**Comment by superxi911** [comment]
LGTM

**Pull request merged**